src/controller/plugin.js

```diff
@@ -11,7 +14,7 @@ export default class Controller {
 
       return response;
     } catch (err) {
-      this.gateway.logger.log("error", "Error:", err);
```

Comment:
**Annotation:** Controllers should not be using gateway's logger, but it's own logger.